applications/test_scheduler/Cargo.toml

```diff
@@ -2,20 +2,18 @@
 name = "test_scheduler"
 version = "0.1.0"
 authors = ["Namitha Liyanage <namithaliyanage@gmail.com>"]
```

Comment:
this is an entirely new crate now, so you can update the authorship to only yourself.